api/src/main/java/org/openmrs/module/eptsreports/metadata/CommonMetadata.java

```diff
@@ -72,12 +73,6 @@ public Concept getPregnantConcept() {
     return getConcept(uuid);
   }
 
-  public Concept getGestationConcept() {
-    String uuid =
-        Context.getAdministrationService().getGlobalProperty("eptsreports.gestationConceptUuid");
```

Comment:
Hi @ningosi, do we no longer  need to use  44  concept?

Reply:
Yes, that concept is no longer required for the question having this concept_id 1982. So anywhere we had that question with an answer as 44, we replace with 1065. By doing that everywhere, I found that concept nolonger being used anywhere, that is why I ended up deleting it. But if there could be a usecase for it, it can be returned, but for me there were no uses needed for it

Reply:
ok thanks Nicholas